src/Http/RequestManager.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Framework3\Http;
+
+use Framework3\Http\Session\Session;
+
+class RequestManager
+{
+    /**
+     * @return Request
+     */
+    public function getFromGlobals(): Request
```

Comment:
Codacy found an issue: [getFromGlobals accesses the super-global variable $_FILES.](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6299285)